addon/mixins/resize-text.js

```diff
@@ -1,13 +1,31 @@
 import Mixin from '@ember/object/mixin';
 import { inject as service } from '@ember/service';
 
+const extractPixels = function(value) {
+  let match = /(\d+)px/i.exec(value || '');
+  return match ? parseInt(match[1], 10) : 0;
+};
+
 export default Mixin.create({
 
   classNames: ['resize-text'],
 
   minSize: 2,
   maxSize: 80,
 
+  // NOTE: shrink and grow are shortcuts (as mentioned below)
+  // the reasoning: often times the template developer does not have access to (or does not know)
+  // the specified font size (in pixels)
+  // obviously that can be computed by you, the developer, but these shortcuts handle it for you
+
+  // passed-in: set to false if you only want to grow the text (never shrink)
+  // setting to false is an alternative to setting maxSize to your current font size
+  shrink: true,
+
+  // passed-in: set to false if you only want to shrink the text (never grow)
+  // setting to false is an alternative to setting minSize to your current font size
```

Comment:
features should be documented in readme, not in code. Please add something like this to README.md instead

Reply:
:+1: